release-notes.txt

```diff
@@ -209,7 +208,6 @@ other issues
 #3366: periph/i2c: handle NACK
 #4488: Making the newlib thread-safe: When calling puts/printf after thread_create(), the CPU hangs
        for DMA enabled uart drivers.
-#4866: periph: GPIO drivers are not thread safe
```

Comment:
This is still open AFAIK...

Reply:
It's a duplicate, see above.

Reply:
You mean [L185](https://github.com/RIOT-OS/RIOT/pull/7917/files#diff-acce5952fb2ed7b16765d3ddab1e2403L186)? Alright then.

Reply:
Yes, though the description doesn't match only with the title of the issue but also with the description:

```
#4866: not all GPIO driver implementations are thread safe: Due to non-atomic operations in the drivers
```